### ISSUE TITLE
Data Augmentation in Soundstream

### ISSUE THREAD

**Issue by cyanbx** [yangdongchao/AcademiCodec#19]
Hi, thanks for your great work. I notice that the `NSynthDataset` used in SoundStream contains data augmentation by adding two audio waveforms, which does not appear in Encodec. I wonder where is this technique proposed, and have you found that it helps the audio quality? Thanks.

**Comment by yangdongchao** [comment]
> Hi, thanks for your great work. I notice that the `NSynthDataset` used in SoundStream contains data augmentation by adding two audio waveforms, which does not appear in Encodec. I wonder where is this technique proposed, and have you found that it helps the audio quality? Thanks.

Yes, it can gain small improvement, but not significant. You can try it.
